client/client-multi/client-c/src/kaa/platform/kaa_client_properties.h

```diff
@@ -24,8 +24,29 @@
 #ifndef POSIX_KAA_CLIENT_PROPERIES_H_
 #define POSIX_KAA_CLIENT_PROPERIES_H_
 
+#include <kaa_error.h>
+
 typedef struct {
-        unsigned long max_update_time;
+    const char *working_directory;
 } kaa_client_props_t;
 
+/**
+ * @brief      Sets the client kaa properties struct by the user-defined struct
+ *
+ * @param[in]  self  Pointer to the user-defined structure
+ * 
+ * @note       since kaa_client_start() is called, memory allocated for kaa_client_props_t
+ *             shouldn't be freed until kaa_client_stop() is called
```

Comment:
state that any of structure fields should bot be changed until `kaa_client_stop()` is called